xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/test/java/org/xwiki/web/DocumentTagsTest.java

```diff
@@ -43,6 +43,7 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.matchesPattern;
 import static org.mockito.Mockito.when;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Imports should be ordered alphabetically within their group. In general, you should use an IDE, ideally Eclipse or IntelliJ and [configure it to use XWiki's code style](https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HConfiguringyourIDEtousetheXWikicodestyle).
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.mockito.Mockito.when;
```